azure-pipelines.yml

```diff
@@ -93,7 +108,7 @@ jobs:
     vmImage: $(macOSVmImage)
     targetFolder: Xamarin.Forms.ControlGallery.Android/legacyRenderers/
     androidProjectArguments: '/t:"Rebuild;SignAndroidPackage" /bl:$(Build.ArtifactStagingDirectory)/android-legacy.binlog'
-    buildConfiguration: $(DefaultBuildConfiguration)
+    buildConfiguration: $(AndroidBuildConfiguration)
```

Comment:
@rmarinho YML Changes look good?

I setup AndroidBuildConfiguration to be a pipeline variable so I can run a release version of CG tests with r8/release/dx set. 

I added this variable to the internal yml and public yml pipelines